src/objects/container/container.stories.mdx

```diff
@@ -0,0 +1,50 @@
+import { Story, Preview, Meta } from '@storybook/addon-docs/blocks';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+import basicDemo from './demo/basic.twig';
+import './demo/styles.scss';
+import './container.scss';
+
+<Meta title="Objects/Container" decorators={[withKnobs]} />
+
+# Container
```

Comment:
Can we add some more explanatory text? What is this pattern intended for? What's the difference between prose and the default? If the default is not for prose, what is it for? That sort of stuff would be helpful.